README.md

```diff
@@ -98,7 +103,7 @@ We follow a systematic Git Workflow -
 
 ### Note
 
-- As the project now uses poetry to manage dependencies, you need to run `poetry add <package_name>` to install the new package.
+- As the project now uses pipenv to manage dependencies, you need to run `pipenv add <package_name>` to install the new package.
```

Comment:
The command is `pipenv install <package_name>`

Reply:
Plus can you add a discord badge also in place of slack?

Reply:
Resolved.